### PR TITLE
completed css-canvas-medical

### PR DESCRIPTION
**Pull request by Wizzo1337**
image
image

### CODE REVIEW

**Comment by Wizzo1337** [comment]
those are 2 screenshots, didn't know how to scroll down while taking a screenshot

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+.description {
+  position: absolute;
+  left: 5%;
+  color: rgb(255, 255, 255);
+  width: 45%;
+  bottom: 28%;
+  font-size: 0.9em;
+  line-height: 1.4;
+}
```
Comment (uzair-ashraf) [inline css-canvas-medical/styles.css:109]: These do not need to be position absolute, they can be done properly with a flex container.

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+button.get-started {
+  position: absolute;
+  bottom: 40px;
+  left: 5%;
+}
```
Comment (uzair-ashraf) [inline css-canvas-medical/styles.css:137]: Same here

File: css-canvas-medical/styles.css (hunk 1 above)
Comment (uzair-ashraf) [reply css-canvas-medical/styles.css:109]: With some margin as well

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]

**Comment by Wizzo1337** [comment]
image
image

File: css-canvas-medical/styles.css [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+.company {
+  position: relative;
+  left: 70%;
+  bottom: 10%;
+}
+
+.product {
+  position: relative;
+  left: 30%;
+}
```
Comment (uzair-ashraf) [inline css-canvas-medical/styles.css:258]: This is not the appropriate use case for position relative.  Margin and padding to adjust movement, with a proper grid system layout will five you the output you are looking for.

File: css-canvas-medical/styles.css [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+  width: 100%;
+  display: flex;
+  height: 60px;
+}
+
+.footer {
+  position: relative;
+  left: 4%;
+  top: 50%;
+}
```
Comment (uzair-ashraf) [inline css-canvas-medical/styles.css:270]: same here, align content center if justify-content start with a flex container will remove the need for position relative.

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]

**Comment by Wizzo1337** [comment]
image
image

**Review by uzair-ashraf — APPROVED** [review]

**Pull request merged**